src/main/java/witchinggadgets/common/items/ItemInfusedGem.java

```diff
@@ -60,6 +60,10 @@ public ItemInfusedGem() {
         this.setCreativeTab(WitchingGadgets.tabWG);
     }
 
+    public static boolean isTeleportationAllowed() {
+        return Config.allowMirrors && !Loader.isModLoaded("dreamcraft");
```

Comment:
Dreamcraft being loaded should really be cached and checked one at boot, instead of queried each time.

Reply:
Absolutely correct

Reply:
Check out Thaumic boots if you want an example on how to do this

Reply:
Surprised we have not asmed into Loader to always cache it internally.